src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -198,6 +202,7 @@ static UCS_CLASS_INIT_FUNC(uct_rc_verbs_iface_t, uct_md_h md, uct_worker_h worke
                                                self->super.config.tx_qp_len);
     self->super.config.tx_moderation = ucs_min(self->super.config.tx_moderation,
                                                self->config.tx_max_wr / 4);
+    self->config.no_fence            = !config->fence;
```

Comment:
why inverse the meaning of the "fence" flag when it's copied to iface? IMHO it's more confusing
let's have just 'self->config.fence'